npm/vite-plugin-cypress-esm/src/index.ts

```diff
@@ -120,21 +118,30 @@ export const CypressEsm = (options?: CypressEsmOptions): Plugin => {
 
     // Ensure import comes at start of line *or* is prefixed by a space so we don't capture things like
     // `Refresh.__hmr_import('')
-    const importRegex = /(?<=^|\s)import (.+?) from (.*)/g
+    const importRegex = /(?<=^|\s)import (.+?) from (['"].*)/g
```

Comment:
We have an import `import { from } from 'myPackage'` that caused the detected import target to be `} from 'myPackage'`. Now, the capturing group has to start with a quote.

Reply:
I further improved the regex to only capture what's between the quotes.